### PR TITLE
update codeowner/author for diagnostics area

### PR DESCRIPTION
**Pull request by sywhang**
## Summary

Updating code owner for diagnostics area

cc @tommcdon @sdmaclea

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -303,7 +303,7 @@
         "docs/core/**/**.md": "dotnetcontent",
         "docs/core/compatibility/**/**.md": "gewarren",
         "docs/core/deploying/**/**.md": "adegeo",
-        "docs/core/diagnostics/**/**.md": "stmaclea",
+        "docs/core/diagnostics/**/**.md": "sywhang",
```
Comment (gewarren) [inline docfx.json:306]:
```suggestion
        "docs/core/diagnostics/**/**.md": "suwhang",
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -72,7 +72,7 @@
 # Deployment
 /docs/core/deploying/ @adegeo
 # Diagnostics
-/docs/core/diagnostics/ @sdmaclea @dotnet/docs
+/docs/core/diagnostics/ @sywhang @sdmaclea @dotnet/docs
```
Comment (sdmaclea) [inline .github/CODEOWNERS:75]: Your welcome to remove my name here if you prefer.

**Review by sdmaclea — APPROVED** [review]

File: docfx.json (hunk 1 above)
Comment (gewarren) [reply docfx.json:306]: Same for the other ones for ms.author (this one is a Microsoft alias).

Comment (sdmaclea) [reply docfx.json:306]: So the interestinb thing is that sometimes these use github user name,  Others times Microsoft user name.  This should be your MS user name.

**Review by gewarren — APPROVED** [review]

**Pull request merged**